test/Concurrency/sendable_keypaths.swift

```diff
@@ -277,3 +277,11 @@ do {
     // expected-warning@-1 {{type 'KeyPath<Foo, Int>' does not conform to the 'Sendable' protocol; this is an error in the Swift 6 language mode}}
   }
 }
+
+public final class TestSetterRef {
+  public internal(set) var v: Int = 0
+
+  public func test1(_ kp: KeyPath<TestSetterRef, Int> = \.v) {} // Ok
+  public func test2(_ kp: KeyPath<TestSetterRef, Int> = \TestSetterRef.v) {} // Ok
+  public func test3(_ kp: KeyPath<TestSetterRef, Int> & Sendable = \.v) {} // Ok
```

Comment:
Could you add test cases verifying that setter accessibility is diagnosed when the type is `WritableKeyPath`?